pentagon/cli.py

```diff
@@ -4,12 +4,21 @@
 import logging
 import traceback
 import pentagon
+import yaml
+import json
+
+from pydoc import locate
+from importlib import import_module
```

Comment:
This does not appear to be used and can be removed. Was this not suitable and instead pydoc is used instead?

Reply:
Correct